sql/setMapSizeKeywords.sql

```diff
@@ -0,0 +1,3 @@
+REPLACE INTO file_keyword(fid,keyword) (SELECT f.fid,'small' FROM file f WHERE cid=2 AND map_width*map_height <= 12*12);
```

Comment:
only relevant once: adding these keywords should go into extract_metadata.py

Reply:
done